mysql-test/suite/component_masking_functions/r/mask_canada_sin.result

```diff
@@ -0,0 +1,42 @@
+#
+# Component Masking Functions: mask_canada_sin
+#
+INSTALL COMPONENT 'file://component_masking_functions';
+SELECT mask_canada_sin("123456789");
+mask_canada_sin("123456789")
+XXXXXXXXX
+SELECT mask_canada_sin("123-456-789");
+mask_canada_sin("123-456-789")
+XXX-XXX-XXX
+SELECT mask_canada_sin(NULL);
+ERROR HY000: mask_canada_sin<string> UDF failed; Wrong argument: must be between 9 and 11 characters
+SELECT mask_canada_sin("");
+ERROR HY000: mask_canada_sin<string> UDF failed; Wrong argument: must be between 9 and 11 characters
+SELECT mask_canada_sin("this is not an canada_sin");
+ERROR HY000: mask_canada_sin<string> UDF failed; Wrong argument: must be between 9 and 11 characters
+SELECT mask_canada_sin("123-456-789", "!");
+mask_canada_sin("123-456-789", "!")
+!!!-!!!-!!!
+SELECT mask_canada_sin("123456789", "!");
+mask_canada_sin("123456789", "!")
+!!!!!!!!!
+# check charset of mask_pan_relaxed()
```

Comment:
Hi, seems like missprint,  mask_canada_sin should be instead of mask_pan_relaxed.